src/main/java/seedu/trackascholar/model/util/SampleDataUtil.java

```diff
@@ -22,22 +22,22 @@ public static Applicant[] getSamplePersons() {
         return new Applicant[] {
             new Applicant(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Scholarship("NUS Global Merit Scholarship"), new ApplicationStatus("pending"),
-                getMajorSet("medicine")),
+                getMajorSet("Medicine")),
             new Applicant(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
                 new Scholarship("NUS Merit Scholarship"),
-                new ApplicationStatus("pending"), getMajorSet("business", "accountancy")),
+                new ApplicationStatus("pending"), getMajorSet("Business", "Accountancy")),
             new Applicant(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
                 new Scholarship("NUS Sports Scholarship"),
-                new ApplicationStatus("accepted"), getMajorSet("physiotherapy")),
+                new ApplicationStatus("accepted"), getMajorSet("Physiotherapy")),
             new Applicant(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
                 new Scholarship("NUS Performing & Visual Arts Scholarship"),
-                new ApplicationStatus("rejected"), getMajorSet("psychology")),
+                new ApplicationStatus("rejected"), getMajorSet("Psychology")),
```

Comment:
Is it possible to have one of the data applicants have no major, just to show that the major field is optional? Might be good to include it, since I saw the JSON file in the test folder also has one.

Reply:
Noted, changes made